satpy/readers/yaml_reader.py

```diff
@@ -322,22 +323,26 @@ def check_file_covers_area(file_handler, check_area):
         return True
 
     def find_required_filehandlers(self, requirements, filename_info):
-        """Find the necessary fhs for the current filehandler.
+        """Find the necessary file handlers for the given requirements.
 
         We assume here requirements are available.
+
+        Raises:
+            KeyError, if no handler for the given requirements is available.
+            RumtimeError, if there is a handler for the given requirements,
```

Comment:
Small typo `Runtime` versus `Rumtime`. I can fix this for you if I don't see anything else that needs changing.